app/translations/src/MapLocaleToLanguageQuery.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+
+import idx from 'idx';
+
+// Copied from grahpql.schema
```

Comment:
Should we update our GraphQL API to be less embarrassing? Or how to keep this up to date?

Reply:
For example, we have now Locale and Language types in the API. What a mess... :-/